packages/app/src/Element/Thread.tsx

```diff
@@ -273,6 +270,13 @@ export default function Thread(props: ThreadProps) {
   const { formatMessage } = useIntl();
   const urlNoteId = location?.pathname.slice(3);
   const urlNoteHex = urlNoteId && bech32ToHex(urlNoteId);
+  const root = useMemo(() => {
+    const currentNote = parsedNotes.find(a => a.Id === urlNoteHex);
+    const rootEventId = currentNote?.Thread?.Root?.Event;
```

Comment:
In some cases the note is replying to a non-root event (perhaps not correctly tagged by clients).

https://fix-broken-note-links.snort-social.pages.dev/e/note12alc8r5hcw42jm57pnxe5pcjt7rrn9jn7aupww6ght0dql88gm9s94p9xn

https://snort.social/e/note12alc8r5hcw42jm57pnxe5pcjt7rrn9jn7aupww6ght0dql88gm9s94p9xn